bin/elao-command-migration

```diff
@@ -3,6 +3,19 @@
 
 require __DIR__ . '/../vendor/autoload.php';
 
+function includeIfExists(string $file): bool
+{
+    return file_exists($file) && include $file;
+}
+
+if (
+    !includeIfExists(__DIR__ . '/../../../autoload.php') &&
+    !includeIfExists(__DIR__ . '/../vendor/autoload.php')
+) {
+    fwrite(STDERR, 'Install dependencies using Composer.'.PHP_EOL);
+    exit(1);
+}
+
 use Elao\CommandMigration\Command\RunCommand;
 use Symfony\Component\Console\Application;
```

Comment:
`use` directives can still be first, it'll not trigger autoloading :)

Reply:
Moved to the top, thanks